salt/preflight/defaults.yaml

```diff
@@ -0,0 +1,27 @@
+mandatory:
+  packages:
+    - util-linux            # provides nsenter, mount
+    - iproute               # provides ip
+    - iptables              # provides iptables
+  sysctl_values:
+    net.bridge.bridge-nf-call-ip6tables: 1
+    net.bridge.bridge-nf-call-iptables: 1
+    net.ipv4.ip_forward: 1
+  available_pathdirs:
+    - /var/lib/etcd
+    - /etc/kubernetes/manifest
+  services:
+    running:
+      - containerd
+recommended:
+  packages:
+    - ebtables              # provides ebtables
+    - ethtool               # provides ethtool
+    - socat                 # provides socat
+    - iproute               # provides tc
```

Comment:
`iproute` is in both `mandatory` as well as `recommended`?

Reply:
yes because tc is recommended and ip is required and both binaries is provided by iproute. So I put it in both sections, but of course, it could be merged

Reply:
OK, can be left here :)